app/util/jira/upload_attachments.sh

```diff
@@ -4,7 +4,7 @@
 ###################    Variables section         ###################
 # Jira version variables
 JIRA_VERSION_FILE="/media/atl/jira/shared/jira-software.version"
-SUPPORTED_JIRA_VERSIONS=(8.0.3 7.13.15 8.5.9)
+SUPPORTED_JIRA_VERSIONS=(8.13.0 8.5.9 8.0.3 7.13.15)
```

Comment:
do not change the order here, this order is used in by `force` flag to identify the latest version.